src/docs/user/references/Notification-service.adoc

```diff
@@ -0,0 +1,257 @@
+=== Overview
+
+The <<_glossary_notification_portal,*Notification Portal*>> allows users to subscribe, be notified and react when different
+types of events occur during <<_glossary_job,*Job*>> execution, on the state of the <<_glossary_proactive_scheduler,*ProActive Scheduler*>> or when
+automating and scheduling jobs with the <<_glossary_job_planner,*Job Planner*>>.
+
+image::notifications-view.png[align="center"]
+
+=== Subscriptions
+
+==== Principle
+
+A notification <<_glossary_notification_subscription,*Subscription*>> consists in a set of events selected by a user for a particular service,
+a type of workflows or a specific workflow.
+
+Event reception is configurable and allows portal notifications, email sending and custom script execution.
+
+A user creating a subscription will need to:
+
+- Select one of the subscription types
+- Select one or more events from a predefined list that user will be notified of.
+- Activate one or more notification methods.
+
+==== Subscription types
+
+There are two types of <<_glossary_notification_subscription,*Subscriptions*>>:
+
+- Service subscriptions such as the Scheduler subscription to be notified when the <<_glossary_proactive_scheduler ,*Scheduler*>> changes its state
+or the Job planner subscription which can be activated with only one event type "All events"
+that corresponds to different issues the <<_glossary_job_planner,*Job Planner*>> may encounter.
+Check out the <<_job_planner_notifications,Job Planner Notifications>> to look at the full list of events.
+- Job subscriptions: To be notified of events that may occur during <<_glossary_workflow,*Workflow*>> executions.
+
+Service subscriptions are unique for each service. They can be enabled or disabled for each particular service.
+
+Job Subscriptions can be multiple, one can create various subscriptions for various kind of workflow types.
+
+By default each user has a Scheduler and a Job Planner subscription.
```

Comment:
remove and improve, think of a dedicated section for "Service subscriptions", where you clarify how to enable or disable them. Similarly a dedicated section on "Job subscriptions" would improve readability.